chrome/background.html

```diff
@@ -0,0 +1,4 @@
+<!doctype html>
+<script src="main.js"></script>
```

Comment:
I'm guessing you're wondering about front-end performance stuff? `main.js` is a super short file, and it ships with the extension, so I don't think there's any need to worry about how it affects browser performance at this point.